transmogrifier/sources/marc.py

```diff
@@ -1,8 +1,200 @@
 import logging
+
+from bs4 import Tag
+
+import transmogrifier.models as timdex
 from transmogrifier.sources.transformer import Transformer
 
 logger = logging.getLogger(__name__)
 
 
 class Marc(Transformer):
     """Marc transformer."""
+
+    def get_optional_fields(self, xml: Tag) -> dict:
+        """
+        Retrieve optional TIMDEX fields from a MARC XML record.
+
+        Overrides metaclass get_optional_fields() method.
+
+        Args:
+            xml: A BeautifulSoup Tag representing a single MARC XML record.
+        """
+        fields: dict = {}
+
+        # alternate_titles
+        alternate_title_marc_data = {
```

Comment:
Given the list comprehension mess I created in my initial `EAD` work, I was going for a balance of readability and abstraction with this config approach but I can see how more complex methods could reduce duplicate code in `get_optional_fields`. Let me know how you see this happening and where you'd like the config data to live

Reply:
That's a lot clearer, thanks!